packages/performance-tests/test/cases/identity-transformer.ts

```diff
@@ -61,24 +64,15 @@ export default async function identityTransformer(iModel: TestIModel, reporter:
     sourceDb.nativeDb.exportSchemas(schemaDumpDir);
     Logger.logInfo(loggerCategory, `dumped schemas to: ${schemaDumpDir}`);
   } finally {
-    const record = {
-      /* eslint-disable @typescript-eslint/naming-convention */
-      "Id": iModel.iModelId,
-      "T-shirt size": iModel.tShirtSize,
-      "Gb size": sizeInGb,
-      "Branch Name": branchName,
-      /* eslint-enable @typescript-eslint/naming-convention */
+    reporterData = {
+      testSuite: "identity transform (provenance)",
+      testName: sourceDb.name,
+      valueDescription: "time elapsed (seconds)",
+      value: entityProcessingTimer?.elapsedSeconds ?? -1,
     };
-    reporter.addEntry(
-      "identity transform (provenance)",
-      `${branchName}: ${iModel.name}`,
-      "time elapsed (seconds)",
-      entityProcessingTimer?.elapsedSeconds ?? -1,
-      record
-    );
     targetDb.close();
     sourceDb.close();
```

Comment:
so we should probably never mutate the source db in test cases and not close it until after all test cases for that iModel have finished. Let's not close it here.

Reply:
let's also try to make sure we open the sourceDb as readonly, the only case I can think of where we may want it writeable is the init fork test where we add our own fed guids to an empty imodel before initialization, and in that one, the test case can manually copy the underlying sqlite file and open the new one as writeable, then discard it at the end of the test.

Reply:
so just to clarify, we want to move open and close into the base test file, and so something like this?
`
testCasesMap.forEach(async (testCase, key) => {...
  sourceDb.open()//readonly
  it(key, async () => {
    reporter = await testCase(sourceDb, sourceBriefcaseArgs, reporter, record);
  }).timeout(0);
  sourceDb.close()
`

Reply:
yes basically, and make sure to add a comment on the `close()` saying that you're doing it to ensure connection cache reusage doesn't affect results.

Reply:
done